src/components/dar_collection_table/DarCollectionTable.js

```diff
@@ -199,6 +207,38 @@ export const DarCollectionTable = function DarCollectionTable(props) {
     This will be updated to account for token based requests on a later ticket
   */
 
+  const updateCollectionIsExpandedById = useCallback((id, val) => {
+    if (collectionsExpandedState[id] !== val) {
+      const newCollectionsExpandedState = cloneDeep(collectionsExpandedState);
+      newCollectionsExpandedState[id] = val;
+      setCollectionsExpandedState(newCollectionsExpandedState);
+    }
+  }, [collectionsExpandedState]);
+
+  const collectionIsExpanded = useCallback((id) => {
+    return collectionsExpandedState[id] === true;
+  }, [collectionsExpandedState]);
+
+  const fetchDarCollection = useCallback((darCollectionId) => {
+    if (!isNil(darCollectionCache[darCollectionId])) {
+      return darCollectionCache[darCollectionId];
+    } else {
```

Comment:
I don't think this is working as intended. When I open/close/open a dar row, the entire collection is re-fetched a second time.

Reply:
Follow-on to this, I can see in the console that the collection is stored in the cache so it looks like there's a lookup problem as opposed to a caching problem.

Reply:
Ah, that makes sense - the duplicate calls looked suspicious.